roles/samdoran.pgsql-replication/tasks/pgsql_version.yml

```diff
@@ -1,13 +1,19 @@
-- name: Get PGSQL version
-  command: psql --version
+- name: Get PGSQL version 10.x
+  command: "{{ pgsqlrep_bash }} psql --version"
```

Comment:
will have to check if we can use this module https://docs.ansible.com/ansible/latest/modules/postgresql_info_module.html#postgresql-info-module to get the version from there.